scripts/prepare_sector_network.py

```diff
@@ -2347,17 +2348,394 @@ def add_biomass(n, costs):
         )
 
 
-def add_industry(n, costs):
-    logger.info("Add industrial demand")
+def add_low_t_industry(n, nodes, industrial_demand, costs, must_run):
+    """
+    Add low temperature heat for industry.
```

Comment:
Would be maybe nice to add the comment here that we assume all industrial demand for biomass is used to serve low temperature demand